StreamChatUITestsAppUITests/Tests/MessageList_Tests.swift

```diff
@@ -341,6 +341,38 @@ final class MessageList_Tests: StreamTestCase {
         }
     }
 
+    func test_editsMessageDecreasingNumberOfTextLines() throws {
```

Comment:
@alteral could these tests be moved closer to the `test_messageIncreases_whenUserEditsMessageWithOneLineText` in this file?